.github/workflows/ci.yml

```diff
@@ -1,18 +1,18 @@
-name: Java CI
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 17
-        uses: actions/setup-java@v2
-        with:
-          java-version: '17'
-          distribution: 'adopt'
-          cache: maven
-      - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify
+name: Java CI

```

Comment:
Please dont push ( and dont change if it is not necessary) files which is not related to homework